doc/source/getting_started/versioning.rst

```diff
@@ -87,7 +87,7 @@ PyMAPDL maintainers greatly appreciate your support and understanding
 during this transition.
 If you have any questions or concerns regarding this change, please
 feel free to reach out to the support team in
-`PyMAPDL discussions <pymapdl_discussions_>`_ or by email |pyansys_contact|.
+`PyMAPDL discussions <pymapdl_discussions_>`_.
```

Comment:
@germa89 I'd recommend chagging to this: If you have any questions or concerns regarding this change, email `pyansys.core@ansys.com <pyansys.core@ansys.com>`_ or post them on the `PyMAPDL Discussions
<https://github.com/ansys/pymapdl>`_ page.

Reply:
Done. Thank you!